explorer/testdata/choice/generic_choice1.carbon

```diff
@@ -0,0 +1,30 @@
+// Part of the Carbon Language project, under the Apache License v2.0 with LLVM
```

Comment:
"generic_choice1" isn't a very descriptive file name. What is this file actually testing that isn't covered by `generic_choice.carbon`?